app/data_mapper_setup.rb

```diff
@@ -0,0 +1,10 @@
+env = ENV['RACK_ENV'] || "development"
+
+DataMapper.setup(:default, "postgres://localhost/bookmark_manager_#{env}")
+
+require_relative './lib/tweet'
+require_relative './lib/user'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+DataMapper.auto_migrate!
```

Comment:
this makes sense and having trouble with an error since changing this and making an association between ```user.rb``` and ```tweet.rb``` that reads ``` undefined method `save_parents' for "Sean":String ``` on ```tweet_spec.rb:9```, i don't really understand what this is?

![screen shot 2015-03-02 at 17 21 59](https://cloud.githubusercontent.com/assets/10243001/6446127/ad711330-c100-11e4-897e-e4dba125b7d2.png)